.github/workflows/grid_client_nightly.yml

```diff
@@ -111,53 +111,33 @@ jobs:
             exit 1
           fi
 
-      - name: Test Results
+      - name: Find issues
+        if: failure()
+        id: find-issues
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
-          echo Dynamic Single Vm: ${{ steps.dynamicsinglevm.outcome }}
-          echo Multiple Vm: ${{ steps.multiplevm.outcome }}
-          echo Kubernetes: ${{ steps.kubernetes.outcome }}
-          echo Vmq Qsfs: ${{ steps.vmqsfs.outcome }}
-          echo Kubernetes Qsfs: ${{ steps.kubernetesqsfs.outcome }}
-          echo Kvstore: ${{ steps.kvstore.outcome }}
-          echo Zdb: ${{ steps.zdb.outcome }}
-          echo Delete all contracts: ${{ steps.deleteall.outcome }}
-
-      - name: Check test script status
-        if: |
-          steps.dynamicsinglevm.outcome != 'success' ||
-          steps.multiplevm.outcome != 'success' ||
-          steps.kubernetes.outcome != 'success' ||
-          steps.vmqsfs.outcome != 'success' ||
-          steps.kubernetesqsfs.outcome != 'success' ||
-          steps.kvstore.outcome != 'success' ||
-          steps.zdb.outcome != 'success' ||
-          steps.deleteall.outcome != 'success'
-        run: exit 1
+          ISSUE_COUNT=$(gh issue list --search "${{github.workflow}} failed during schedule on ${{ env.NETWORK }}" --json title --jq '. | length')
+          echo "issue_count=$ISSUE_COUNT" >> $GITHUB_ENV
 
       - name: Create GitHub Issue on Failure
-        if: failure() && '$(date +%u)' != '5' && '$(date +%u)' != '6'
+        if: failure() && env.issue_count == '0'
         uses: dacbd/create-issue-action@main
         with:
           token: ${{ github.token }}
-          title: |
-            ${{ github.workflow }} failed during ${{ github.event_name }} on network ${{ env.NETWORK }}
+          title: ${{github.workflow}} failed during schedule on ${{ env.NETWORK }}
```

Comment:
Notice that, you have changed the title, is that mean the old issues with the old title will be ignored?

Reply:
I don't search for the exact issue title but I use a search flag like the one used in github. Old issues will be included in the result too.